main/templates/team/base.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
This structure is good. In future how team page looks and how our home page for the app look may differ.  For now move the base.html to "./main/templates/index.html" so that other templates too can use this besides team templates.